docs/content/envoy-performance.md

```diff
@@ -0,0 +1,164 @@
+Authorization
+API](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ext_authz_filter.html) enabled and the
+`RBAC` policy described above loaded into OPA. All values are in microseconds.
+
+##### OPA Evaluation
+
+`OPA Evaluation` is the time taken to evaluate the policy.
+
+| Number of Requests per sec | 75% | 90% | 95% | 99% | 99.9% | 99.99% | Mean | Median |
+|--------------------------|---|---|---|---|-----|------|----|------|
+| `10` | `467.122` | `571.028` | `579.272` | `579.272` | `579.272` | `579.272` | `437.282` | `437.367` |
+| `100` | `374.714` | `439.620` | `502.146` | `736.478` | `737.025` | `737.025` | `326.840` | `307.902` |
+| `1000` | `251.582` | `386.282` | `526.037` | `1513.457` | `5627.604` | `5629.248` | `269.490` | `202.471` |
+| `2000` | `223.174` | `463.464` | `1461.558` | `3456.464` | `8251.585` | `8289.024` | `345.520` | `168.145` |
+| `3000` | `253.609` | `660.672` | `2171.718` | `6878.808` | `149920.172` | `152030.931` | `800.997` | `163.617` |
+| `4000` | `266.848` | `625.534` | `1957.568` | `18970.466` | `152209.408` | `153406.983` | `1010.549` | `162.288` |
+| `5000` | `302.344` | `1153.263` | `2283.017` | `6149.344` | `131990.115` | `132097.885` | `984.640` | `170.703` |
+
+##### gRPC Server Handler
+
+`gRPC Server Handler` is the total time taken to prepare the input for the policy, evaluate the policy (`OPA Evaluation`)
+and prepare the result.
+
+| Number of Requests per sec | 75% | 90% | 95% | 99% | 99.9% | 99.99% | Mean | Median |
+|--------------------------|---|---|---|---|-----|------|----|------|
+| `10` | `919.938` | `1156.589` | `1181.885` | `1181.885` | `1181.885` | `1181.885` | `766.204` | `732.136` |
+| `100` | `681.010` | `777.223` | `857.337` | `1543.975` | `1544.085` | `1544.085` | `580.151` | `541.035` |
+| `1000` | `430.083` | `622.528` | `1070.177` | `3000.366` | `6792.175` | `6793.221` | `461.363` | `346.520` |
+| `2000` | `386.011` | `992.878` | `2292.579` | `5387.087` | `8935.611` | `8951.691` | `552.703` | `274.954` |
+| `3000` | `458.886` | `2169.441` | `4609.724` | `48793.693` | `296123.974` | `299798.509` | `2364.095` | `270.948` |
+| `4000` | `523.895` | `3168.597` | `8197.937` | `127324.217` | `210427.167` | `212126.793` | `3990.733` | `274.938` |
+| `5000` | `556.581` | `2442.139` | `3943.508` | `9178.572` | `303796.678` | `309531.296` | `1283.640` | `294.794` |
+
+In the analysis so far, the gRPC client used in Envoy's External authorization filter configuration is the [Google C++ gRPC client](https://github.com/grpc/grpc).
+The following graph displays the latency distribution for the same four conditions described previously (ie. *App Only*,
+*App and Envoy*, *App, Envoy and OPA (NOP policy)* and *App, Envoy and OPA (RBAC policy)*) by sending *100 requests/sec*
+to the sample application but now using Envoy’s in-built gRPC client.
+
+{{< figure src="hist-envoy-grpc-100.png" width="200" >}}
```

Comment:
Personally, I find these a little hard to read -- the font size is a bit smaller than the surrounding text. But this is no blocker now, I guess, we could add a backlog item to fix that. (And perhaps have some shiny reactive plot with nice actions on mouseover...? 😉)


![image](https://user-images.githubusercontent.com/870638/108361886-5612db00-71f3-11eb-9f71-a9d751d22f68.png)